CheckoutTests/Stubs/StubCalendar.swift

```diff
@@ -9,12 +9,23 @@ import Foundation
 @testable import Checkout
 
 final class StubCalendar: CalendarProtocol {
+  var currentDateToReturn: Date?
+
+  func current() -> Date {
+    currentDateToReturn ?? Date()
+  }
+
+  var forceToReturnNilComponents: Bool = false
   var dateFromComponentsToReturn: Date?
   private(set) var dateFromComponentsCalledWith: DateComponents?
 
   func date(from components: DateComponents) -> Date? {
```

Comment:
Is this a helper method should this not be part of the actual code itself than the Unit tests ?

Reply:
do you mean `func date(from components: DateComponents) -> Date?` ? @deepesh-vasthimal-cko

Reply:
Yes

Reply:
It's a necessity of `CalendarProtocol` and has been already there and been used to stub the response of it @deepesh-vasthimal-cko